<?xml version="1.0" encoding="UTF-8"?>
<testsuite errors="0" failures="0" file="test_valid_fail_eq_bytes.py" name="test_valid_fail_eq_bytes.TestValidFailEqBytes-20191124111326" skipped="0" tests="1" time="0.000" timestamp="2019-11-24T11:13:26">
	<testcase classname="test_valid_fail_eq_bytes.TestValidFailEqBytes" file="spec/python/test_valid_fail_eq_bytes.py" line="9" name="test_valid_fail_eq_bytes" time="0.000" timestamp="2019-11-24T11:13:26">
		<system-out>
<![CDATA[]]>		</system-out>
		<system-err>
<![CDATA[]]>		</system-err>
	</testcase>
</testsuite>
